client/src/pages/Demo.js

```diff
@@ -0,0 +1,7 @@
+import React from "react";
+
+const Demo = () => {
```

Comment:
Do we need this Demo component if we have the one in the Demo folder?

Reply:
sorry, looks like i overlooked this question. i don't think so, i think we just need one demo page

Reply:
Do I need to fix anything on this PR? Just following up

Reply:
This looks good to me, thank you @rafaover